manifests/buckets.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: black_flake
+spec:
+  generateBucketName: black_flake-
+  storageClassName: ocs-storagecluster-ceph-rgw
+  additionalConfig:
+    maxObjects: "1000"
+    maxSize: "2G"
```

Comment:
Is it possible to get more than 2gb? We might run into resource issues.

Reply:
Let's start with the default to get moving and then see.

Reply:
you can request more... the default values in `additionalConfig` is taken from Rook docs, it's not even "the default" for the cluster. Request what you need. :)

Reply:
After consultation with @tumido, I have increased the request to 4gb.

@tumido : what happens if we change the claim? Will the data persist or will it be deleted?

Reply:
I have no idea. I've tried updating the `ObjectBucketClaim` spec but it did not propageted to `ObjectBucket` resource. I think the bucket will stay intact and the quota is not updated. This may need further research and reaching up to upstream. I don't think this is supporter right now.

Reply:
https://github.com/operate-first/support/issues/75